### ISSUE TITLE
`@DisableFastJsonCircularReferenceDetect` not effective when working with `@PropertySpec`

### ISSUE THREAD

**Issue by huxudong** [actframework/actframework#126]
`@DisableFastJsonCircularReferenceDetect` doesn't work in the following code:

```java
    @DisableFastJsonCircularReferenceDetect
    @act.util.PropertySpec("uuid,title,tag,module.name")
    public List<T> list(){}
```

The workaround is separate `@DisableFastJsonCircularReferenceDetect` and `PropertySpec`:

```java
    @JSONField(serialzeFeatures = SerializerFeature.DisableCircularReferenceDetect)
    private User user;

    @act.util.PropertySpec("uuid,title,tag,module.name")
    public List<T> list(){}
```

**Comment by greenlaw110** [comment]
@huxudong I've pushed a fix to `1.3.0-126-SNAPSHOT`. can you please help to verify the fix? Please do not close the issue if it passed. Just drop a mesage here.

Please also help to test other JSON output scenario, e.g. with `@PropertySpec` or  `@DisableFastJsonCircularReferenceDetect` separately and check if the fix breaks those scenarios.

Thanks ...

**Comment by greenlaw110** [comment]
@huxudong Actually just found this from the source code:

```
// Note: we can't check DisableFastJsonCircularReferenceDetect here because if
// that option is set, then FastJson will skip the JsonSerializer.context setting
// and there is property filter mechanism will break
```

Meaning we just can't use `@DisableFastJsonCircularReferenceDetect` along with `@PropertySpec` for the moment
